docs/Reference/Reply.md

```diff
@@ -674,6 +674,15 @@ fastify.get('/streams', function (request, reply) {
   reply.send(stream)
 })
 ```
+When using async-await you will need to return the reply object:
```

Comment:
@climba03003, both options returns Promise, the difference only is what inside - nothing or reply object, both of them handled same way and this part confuses me a bit - why I need to return reply if I can just await it?

```suggestion
When using async-await you will need to return or await the reply object:
```

Reply:
The difference is just `implicit` or `explicit` handle of `Promise`.
`return` or `await` both `await`ing `reply` in the end.

The `await` here is used to signal the `payload` is not ready yet if you do something off the track.